cookies/samesite/get_all_cookies-default-samesite.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>TestDriver get_all_cookies method w/ default SameSite</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script>
+promise_test(async t => {
+  document.cookie = "test0=0";
+  const cookies = await test_driver.get_all_cookies();
+  assert_equals(cookies.length, 1);
+  let cookieMap = new Map();
```

Comment:
Given the above assert, this can be simplified to just grab `cookies[0]`, perhaps as:

```suggestion
  const cookie = cookies[0];
```

Reply:
I'll save this for a follow up so the repos can be in sync first